lxd/firewall/drivers/drivers_xtables.go

```diff
@@ -936,6 +936,19 @@ func (d Xtables) InstanceSetupProxyNAT(projectName string, instanceName string,
 
 	comment := d.instanceDeviceIPTablesComment(projectName, instanceName, deviceName)
 
+	targetPortRanges := PortRangesFromSlice(forward.TargetPorts)
+	for _, targetPortRange := range targetPortRanges {
+		targetPortStr := portRangeStr(targetPortRange, ":")
```

Comment:
Could we call this targetPortRangeStr or targetPortsStr to make it clear it could contain multiples.